cv-data-service-library/src/main/java/com/trihydro/library/helpers/CdotGisConnector.java

```diff
@@ -0,0 +1,56 @@
+package com.trihydro.library.helpers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import com.trihydro.library.service.RestTemplateProvider;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+
+@Component
+public class CdotGisConnector {
+  private final String baseUrl = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded";
+
+  private final RestTemplateProvider restTemplateProvider;
+
+  private final Logger logger = LoggerFactory.getLogger(CdotGisConnector.class);
+
+  @Autowired
```

Comment:
We should be able to remove this autowired since there's only one constructor

Reply:
Removed